.travis.yml

```diff
@@ -8,6 +8,7 @@ script:
 
 after_success:
   - npm run codecov
+  - npm run nsp
```

Comment:
Counter proposal (if we want to trade 2-3 seconds per build for some smaller diff churn).
Instead of running `npm run nsp` here, we can use npm 5's `npx` magic to just call `npx nsp check` here, then not have to touch package.json or package-lock.json.

**Pros:**
- Moves check purely to Travis or Circle CI.

**Cons:**
- Makes it a bit more challenging to run `nsp` check locally (unless you install it globally or do similar `npx` trick).
- Installs the `nsp` module "on-demand" each time the CI runs, so in addition to the automatic `npm install`, the packages would need to be downloaded again for each build and not get cached in package-lock.json or shrinkwrap or yarn.lock files.